src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,27 @@
+package tasks;
+
+public class Task {
+    protected String name;
+    protected boolean done;
+
+    public Task(String input){
+        this.name = input;
+        this.done = false;
+}
+    public void taskDone(){
+
```

Comment:
There is inconsistent spacing within methods in this file. Here you have left a empty line before the return statement.  In `getName()`, there is no empty line before return statement. You could remove empty lines to improve readability.